asmcomp/arm/proc.ml

```diff
@@ -109,6 +109,8 @@ let stack_slot slot ty =
 
 (* Calling conventions *)
 
+let size_domainstate_args = 64 * 8
```

Comment:
```suggestion
let size_domainstate_args = 64 * 4
```

Reply:
Well spotted.  I put `64 * size_int` for all ports, it's clearer and less error prone.